Author: Daniel Baumann <daniel@debian.org>
Description: Sets the gnulib directory.

Index: gnulib/gnulib-tool
===================================================================
--- gnulib.orig/gnulib-tool	2011-12-11 06:54:46.000000000 +0000
+++ gnulib/gnulib-tool	2011-12-11 06:55:12.000000000 +0000
@@ -455,7 +455,7 @@
       * ) self_abspathname=`echo "$self_abspathname" | sed -e 's,/[^/]*$,,'`/"$linkval" ;;
     esac
   done
-  gnulib_dir=`echo "$self_abspathname" | sed -e 's,/[^/]*$,,'`
+  gnulib_dir=/usr/share/gnulib
 }
 
 # func_tmpdir
