Author: Daniel Baumann <daniel@debian.org>
Description: Adjusts shebang.

Index: gnulib/lib/uniname/gen-uninames.lisp
===================================================================
--- gnulib.orig/lib/uniname/gen-uninames.lisp	2011-02-17 23:11:25.000000000 +0000
+++ gnulib/lib/uniname/gen-uninames.lisp	2011-02-17 23:11:38.000000000 +0000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/clisp -C
+#!/usr/bin/clisp -C
 
 ;;; Creation of gnulib's uninames.h from the UnicodeData.txt table.
 ;;; Bruno Haible 2000-12-28
